nidirect_workflow/src/Controller/AuditController.php

```diff
@@ -101,6 +116,7 @@ public function confirmAudit($nid) {
       $message = "nid " . $nid . " " . $this->t("has been audited by") . " ";
       $message .= $this->account->getAccountName() . " (uid " . $this->account->id() . ")";
       $this->logger->notice($message);
+      \Drupal::messenger()->addMessage(t('Content has been marked as audited'));
```

Comment:
The messenger service should be injected into the controller as a dependency, also of note is: https://www.drupal.org/node/2928994

Reply:
Wow, you're on the ball this morning - I hadn't even requested another review yet !

Good point, but looking through core it seems that I don't have to inject the messenger as it is already available in ControllerBase, so have just changed it to:

$this->messenger()->addMessage

Reply:
haha, that's true, good catch.